tests/contrib/psycopg/test_psycopg.py

```diff
@@ -308,16 +260,15 @@ def test_composed_query_identifier(self):
                 dict(name="postgres.query", resource=query.as_string(db)),
             )
 
-    @snapshot()
-    @skipIf(PSYCOPG2_VERSION < (2, 7), "SQL string composition not available in psycopg2<2.7")
+    @snapshot(async_mode=True)
     def test_composed_query_encoding(self):
```

Comment:
Does the fact that there's no new snapshot test for this mean that we get the same snapshot for psycopg2 and psycopg3? Just want to make sure we ran it and confirmed that.